R/plotAbundanceDensity.R

```diff
@@ -0,0 +1,171 @@
+#' Plot abundance density
+#'
+#' This function plots abundance of the most abundant taxa. 
+#'
+#' @param object a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'    object.
+#'
+#' @param abund_values a single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}} to be
+#'   plotted. (default: \code{abund_values = "counts"})
+#'
+#' @param n a positive integer specifying the number of the most abundant taxa to show. 
+#'  
+#' @param colour_by a single character value defining a column from \code{colData}, that is used to
+#'   color plot. Must be a value of \code{colData()} function.
+#'   
+#' @param ... additional parameters for plotting. See 
+#'   \code{\link{mia-plot-args}} for more details
+#'
+#' @details
+#' This function plots abundance of the most abundant taxa. Abundance is plotted as
+#' a point plot, where x-axis represents abundance and y-axis taxa. Each point represents
+#' abundance of individual taxa in individual sample. Most common abundances are shown
```

Comment:
Make sure, that the man page picks up all the options and describes them clearly. Especially if option are disregarded because of other options (e.g. the layout option), it should be spelled out specifically. 

Here you have to add the `layout` option directly and not show them in the seperate `...` plot-args man page.